instrumentation/play/play-mvc/play-mvc-2.6/javaagent/build.gradle.kts

```diff
@@ -60,19 +60,21 @@ testing {
   }
 }
 
+val testLatestDeps = findProperty("testLatestDeps") as Boolean
 tasks {
-  if (findProperty("testLatestDeps") as Boolean) {
+  if (testLatestDeps) {
     // disable regular test running and compiling tasks when latest dep test task is run
     named("test") {
       enabled = false
     }
-    named("compileTestGroovy") {
-      enabled = false
-    }
+  }
 
-    check {
-      dependsOn(testing.suites)
-    }
+  named("latestDepTest") {
+    enabled = testLatestDeps
+  }
```

Comment:
why is this needed?

Reply:
it wasn't triggering latestDepTest module test when i have -PtestLatestDeps=true locally.

Reply:
you sure about that? seems to work fine for me, Perhaps nothing was run because you had already run the tests and got the results from build cache?

Reply:
let me 'invalidate cache' and try it without this.